src/screens/AccountDetails.js

```diff
@@ -125,7 +125,7 @@ This account can only be recovered with its associated recovery phrase.`,
                 { value: 'AccountEdit', text: 'Edit' },
                 { value: 'AccountPin', text: 'Change Pin' },
                 { value: 'AccountBackup', text: 'View Recovery Phrase' },
-                { value: 'AccountDelete', text: 'Delete', textStyle: styles.deleteText }]}
+                { value: 'AccountDelete', text: 'Delete & Archive Account', textStyle: styles.deleteText }]}
```

Comment:
idem here, I think archive is enough.
```suggestion
                { value: 'AccountDelete', text: 'Archive', textStyle: styles.deleteText }]}
```